packages/language-server/package.json

```diff
@@ -39,14 +39,17 @@
     "@xml-tools/parser": "1.0.7",
     "assert-never": "1.2.1",
     "fs-extra": "9.0.1",
+    "get-proxy": "^2.1.0",
     "lodash": "4.17.21",
     "node-fetch": "2.6.1",
+    "simple-proxy-agent": "^1.1.0",
     "vscode-languageserver": "6.1.1",
     "vscode-languageserver-textdocument": "1.0.1",
     "vscode-uri": "2.1.2"
   },
   "devDependencies": {
-    "@types/node-fetch": "2.5.10",
+    "@types/http-proxy-agent": "^4.0.1",
```

Comment:
how come to **types** dev-dep does not match any of the names of the new dependencies ?

Reply:
removed the http proxy type, that was from my previous proxy attemt, forgot to remove the dependency